.travis.yml

```diff
@@ -0,0 +1,24 @@
+# @Date:   2017-03-20T16:36:18-04:00
+# @Project: scatt-jacc-caj
+# @Filename: travis.yml
+# @Last modified time: 2017-03-20T16:53:05-04:00
+
+#defining the language used
+language: java
+
+script:
+#installing dependencies
+- ./gradlew assemble
+#calling default script
+- ./gradlew check
+
+
+#the following lines help travis avoid uploading of unneccessary caches related to

```

Comment:
I found them on TravisCI's java development web page.
Here's a link, check it out:
https://docs.travis-ci.com/user/languages/java/#Projects-Using-Gradle